cypress/e2e/08_add_monument/add_monument_workflow.cy.js

```diff
@@ -0,0 +1,631 @@
+describe('Going through the Add Monument Workflow', function () {
+
+    beforeEach(() => {
+        cy.login();
+        cy.visit('/plugins/init-workflow');
+    });
+
+    it('Go through the workflow and populate all fields', function () {
+        cy.contains('Workflows');
+        cy.contains('Add Monument').click();
+        cy.wait(2000);
+
+        // Add Monument 1st page
+        cy.get('[aria-label="Start new or please select from below"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').contains('HA/01').click();
+        cy.get('.btn-success').contains('Start New').click();
+        cy.wait(2000);
+
+        // Start tab
+        cy.wait(4000);
+        cy.get('.card_component.resourceid').contains('HA Number');
+        cy.get('.form-control').should('be.disabled');
```

Comment:
Proper disabled check, there should also be a check here that this field is actually populated